### ISSUE TITLE
Searching by email address for a user does not work

### ISSUE THREAD

**Issue by hermanschutte** [ForestAdmin/forest-laravel#37]
When searching for a user in the User model view I get the following error:

```
FatalThrowableError in ResourcesGetter.php line 147:
Call to a member function getFields() on null
in ResourcesGetter.php line 147
at ResourcesGetter->addSearch(object(Builder)) in ResourcesGetter.php line 39
at ResourcesGetter->ForestAdmin\ForestLaravel\Http\Services\{closure}(object(Builder)) in Builder.php line 197
at Builder->where(object(Closure)) in ResourcesGetter.php line 39
at ResourcesGetter->perform() in ResourcesController.php line 23
at ResourcesController->index(object(Request), 'user')
at call_user_func_array(array(object(ResourcesController), 'index'), array(object(Request), 'user')) in Controller.php line 55
at Controller->callAction('index', array(object(Request), 'user')) in ControllerDispatcher.php line 44
at ControllerDispatcher->dispatch(object(Route), object(ResourcesController), 'index') in Route.php line 203
at Route->runController() in Route.php line 160
at Route->run() in Router.php line 559
at Router->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 30
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in AuthenticateForestUser.php line 28
at AuthenticateForestUser->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in HandleCors.php line 42
at HandleCors->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Router.php line 561
at Router->runRouteWithinStack(object(Route), object(Request)) in Router.php line 520
at Router->dispatchToRoute(object(Request)) in Router.php line 498
at Router->dispatch(object(Request)) in Kernel.php line 174
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request)) in Pipeline.php line 30
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in RobotsMiddleware.php line 14
at RobotsMiddleware->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 46
at CheckForMaintenanceMode->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Kernel.php line 149
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 116
at Kernel->handle(object(Request)) in index.php line 53
at require('/Users/hermanschutte/Development/Sites/xxx/public/index.php') in server.php line 133
```

**Comment by arnaudbesnier** [comment]
Thanks for the report, I'll look into it today.

**Comment by arnaudbesnier** [comment]
The fix has just been released in the liana version 0.0.20
